docs/components/tabs.md

```diff
@@ -192,166 +216,124 @@ Here's an example (code only) of utlizing a dynamic `router-view` component with
     </router-link>
   </template>
 </KTabs>
-<router-view
-  v-slot="{ route }"
->
+
+<router-view v-slot="{ route }">
   <h3>Router View content</h3>
   <p>{{ route.path }}{{ route.hash }}</p>
 </router-view>
 ```
 
 ## Slots
 
-### Anchor Content
+### anchor & panel
 
 The tab control defaults to the `tab.title` string. You may use the `#{tab.hash}-anchor` slot to customize the content of the tab control.
 
+In order provide the tab panel content (when the `hasPanels` prop is set to `true`) you must slot the content in the named slot, defined by the `tab.hash` string, without the `#`.
```

Comment:
```suggestion
In order provide the tab panel content (when the `hasPanels` prop is set to `true`) you must slot the content into the corresponding named slot, defined by the `tab.hash` string, without the `#`.
```

Reply:
Can you add another sentence here as an example, e.g. "If the `tab.hash` is `#name-tab` the slot name would be `name-tab`"

... or whatever you think a good way to explain it would be